### ISSUE TITLE
Spotcast error

### ISSUE THREAD

**Issue by prbtt** [fondberg/spotcast#348]
Logger: spotipy.client
Source: custom_components/spotcast/spotcast_controller.py:281
Integration: Spotcast (documentation, issues)
First occurred: 09:37:04 (4 occurrences)
Last logged: 09:52:23

HTTP Error for PUT to https://api.spotify.com/v1/me/player/play?device_id=7964df4f-7ca0-4117-a3c2-c81c48b5770a_amzn_2 with Params: {} returned 401 due to Permissions missing

Automation 

```yaml
alias: Campainha
description: ""
trigger:
  - type: turned_on
    platform: device
    device_id: ************************
    entity_id: binary_sensor.campainha
    domain: binary_sensor
condition: []
action:
  - service: spotcast.start
    data:
      start_volume: 100
      random_song: true
      device_name: Área nobre
      uri: spotify:playlist:3TVO9zfng4fRfWsKBxEXoa
  - delay:
      hours: 0
      minutes: 0
      seconds: 45
      milliseconds: 0
  - service: media_player.media_pause
    data: {}
    target:
      entity_id: media_player.spotify_rodolfo_pamplona_filho
mode: single
```

**Comment by engeles20** [comment]
Might want to delete your device id

**Comment by fcusson** [comment]
closed as this is a duplicate for #347 as mentionned by hmmbob
